### DISCUSSION TITLE
How to get rid of black triangle character in leading and in other positions in themes?

### DISCUSSION THREAD

**Discussion by carpet92** [JanDeDobbeleer/oh-my-posh#2555] [Troubleshoot]
Agnoster:

leading triangle

![Screenshot (676)](https://user-images.githubusercontent.com/20873402/180114328-23ec58eb-3f5e-4fce-9825-fb27a1a71810.png)

Atomic:

triangle character at the end

![Screenshot (677)](https://user-images.githubusercontent.com/20873402/180114335-e64a51c4-9e24-45af-a2f8-b49d56662298.png)

How to get rid of it? What change I should make to `theme-name.omp.json` file to make triangle disappeared? According to `Atomic` theme it looks like it should be clearly transparent but appears as triangle.

Setup:

```
Windows Terminal Preview 1.15.1862.0 
PowerShell 7.2.5
PSReadLine 2.2.6
oh-my-posh 8.20.1
Hack Nerd Font (same happens with CaskaydiaCove Nerd Font)

Terminal-Icons 0.10.0
```

$PROFILE:

```
clear
oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\agnoster.omp.json" | Invoke-Expression
Import-Module Terminal-Icons
New-Alias code code-insiders
```

settings.json - https://pastebin.com/raw/pUghg5QZ

**Comment by JanDeDobbeleer** [comment]
You need to update Windows Terminal, that was a bug on their end.

**Reply by carpet92** [reply]
But it is already latest preview version: [Windows Terminal Preview v1.15.186](https://github.com/microsoft/terminal/releases/tag/v1.15.1862.0)

**Reply by carpet92** [reply]
The issue isn't reproduced in your terminal?

**Reply by JanDeDobbeleer** [reply]
@Zx-EvM it used to be yes, can also depend on the renderer being used in Windows Terminal. It's a bug on their end as it only happens on WT, and they added this fix to their changelog a few weeks ago:

> We no longer suppress black blackground or gray foreground for PowerShell (#13352)
We have chosen to remove this workaround as newer versions of PowerShell's PSReadline component contain a fix for the issue.
This was a compatibility band-aid that was impacting the capabilities of great projects such as Oh My Posh.
❗ If you see unexpected black backgrounds appearing behind text while typing a command in PowerShell, make sure your PSReadline version is up to date. You can update your version of PSReadline by running the command, Update-Module PSReadline.

**Reply by carpet92** [reply]
@JanDeDobbeleer it's surely fixed the https://github.com/microsoft/terminal/issues/13513 but triangle is still appears. Even after I installed the latest PSReadLine. Oh, anyway I decided to use `amro` theme which is don't have any glyphs. So, this is the only working fix

**Reply by JanDeDobbeleer** [reply]
@Zx-EvM I would ask the Windows Terminal team to be honest, maybe it has to do with another setting on your end.

**Reply by carpet92** [reply]
@JanDeDobbeleer I've asked the terminal team on this issue and got some encouraging answer. I'll just leave it here: https://github.com/microsoft/terminal/discussions/13553#discussioncomment-3201718
